content/lessons/algolia-cloud-functions/index.md

```diff
@@ -146,7 +146,7 @@ firebase deploy --only functions
 
 ## Optional: Seed the Database
 
-In the video you saw how I quickly seeded Firestore with dummy data, which was accomplished with [Faker](https://github.com/marak/Faker.js/) and the Firebase Admin SDK. Create a vanilla node script in the functions directory that sends writes to the database. 
+In the video you saw how I quickly seeded Firestore with dummy data, which was accomplished with Faker and the Firebase Admin SDK. Create a vanilla node script in the functions directory that sends writes to the database. 
```

Comment:
Instead of removing the link let's just update the existing one to https://github.com/faker-js/faker